src/web/app/src/components/SignUp/Forms/Review.tsx

```diff
@@ -125,16 +127,18 @@ const Review = connect<{ accountError: string | undefined }, SignUpForm>((props)
           </div>
           <div>
             <div className={classes.blogRss}>
-              <h3 className={classes.titleRss}>Blog RSS:</h3>
+              <h3 className={classes.titleRss}>Blog RSS Snippet:</h3>
```

Comment:
I don't think you need this change, since it will show the full-text sometimes, and other times cut it off, but it will always be a URL.

Reply:
Sure!